src/LibMulticaller.sol

```diff
@@ -66,31 +52,35 @@ library LibMulticaller {
      *      Otherwise, returns `msg.sender`.
      */
     function sender() internal view returns (address result) {
-        /// @solidity memory-safe-assembly
-        assembly {
-            mstore(0x00, caller())
-            let withSender := MULTICALLER_WITH_SENDER
-            if eq(caller(), withSender) {
-                if iszero(staticcall(gas(), withSender, codesize(), 0x00, 0x00, 0x20)) {
-                    revert(codesize(), codesize()) // For better gas estimation.
-                }
-            }
-            result := mload(0x00)
-        }
+        return getAt(MULTICALLER_WITH_SENDER);
     }
 
     /**
      * @dev Returns the caller of `aggregateWithSigner` on `MULTICALLER_WITH_SIGNER`,
      *      if the current context's `msg.sender` is `MULTICALLER_WITH_SIGNER`.
      *      Otherwise, returns `msg.sender`.
      */
-    function signer() internal view returns (address result) {
+    function signer() internal view returns (address) {
+        return getAt(MULTICALLER_WITH_SIGNER);
+    }
+
+    function multicallerGetAt(address loc) internal view returns (address result) {
+        /// @solidity memory-safe-assembly
+        assembly {
+            mstore(0x00, 0x00)
+            if iszero(staticcall(gas(), loc, codesize(), 0x00, 0x00, 0x20)) {
+                revert(codesize(), codesize()) // For better gas estimation.
+            }
+            result := mload(0x00)
+        }
+    }
+
+    function getAt(address loc) internal view returns (address result) {
```

Comment:
I'll rename to `get(address a)`, `getIfCallerIs(address a)` respectively, for mouthfeel.